kura/org.eclipse.kura.camel.cloud.factory/src/org/eclipse/kura/camel/cloud/factory/internal/CamelManager.java

```diff
@@ -180,7 +189,8 @@ public Tocd getDefinition() {
         final Set<String> ids = new TreeSet<>();
         try {
 
-            final Collection<ServiceReference<CamelFactory>> refs = FrameworkUtil.getBundle(CamelManager.class).getBundleContext().getServiceReferences(CamelFactory.class, null);
+            final Collection<ServiceReference<CamelFactory>> refs = FrameworkUtil.getBundle(CamelManager.class)
```

Comment:
We discussed yesterday about the required changes for this PR, we probably will find all kinds of other things we could to in addition. But let's finish this up, it is open for two weeks already.

Beside that, I can't that class in the Kura setup. Maybe you can point me to it.